# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webauthn/WebAuthnTransactionChild.h
# Commit: eb8cc8d7badf
# Full Hash: eb8cc8d7badf8643681a35d38d88c751e08926fa
# Author: John Schanck <jschanck@mozilla.com>
# Date: 2025-02-11 09:01:30
# Regressor Bug: 1945969
# File Overlap Count: 1
# Description:
#   Bug 1945969 - part 5: rename WebAuthnManager to WebAuthnHandler. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D237257
# ==============================================================================

diff -r add154b24d33 -r eb8cc8d7badf dom/webauthn/WebAuthnTransactionChild.h
--- a/dom/webauthn/WebAuthnTransactionChild.h	Mon Feb 10 20:11:06 2025 +0000
+++ b/dom/webauthn/WebAuthnTransactionChild.h	Mon Feb 10 20:11:06 2025 +0000
@@ -12,13 +12,13 @@
 /*
  * Child process IPC implementation for WebAuthn API. Receives results of
  * WebAuthn transactions from the parent process, and sends them to the
- * WebAuthnManager either cancel the transaction, or be formatted and relayed to
+ * WebAuthnHandler either cancel the transaction, or be formatted and relayed to
  * content.
  */
 
 namespace mozilla::dom {
 
-class WebAuthnManager;
+class WebAuthnHandler;
 
 class WebAuthnTransactionChild final : public PWebAuthnTransactionChild {
  public:
@@ -28,13 +28,12 @@
 
   void ActorDestroy(ActorDestroyReason why) override;
 
-  void SetManager(WebAuthnManager* aMananger);
+  void SetHandler(WebAuthnHandler* aMananger);
 
  private:
   ~WebAuthnTransactionChild() = default;
 
-  // Nulled by ~WebAuthnManager() when disconnecting.
-  WebAuthnManager* mManager;
+  WebAuthnHandler* mHandler;
 };
 
 }  // namespace mozilla::dom